boards/st/stm32h745i_disco/doc/index.rst

```diff
@@ -0,0 +1,265 @@
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| QSPI NOR  | on-chip    | off-chip flash                      |
++-----------+------------+-------------------------------------+
+| FDCAN     | on-chip    | fdcan                               |
++-----------+------------+-------------------------------------+
+| FMC       | on-chip    | memc (SDRAM)                        |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not yet supported on this Zephyr port.
+
+The default configuration per core can be found in the defconfig files:
+:zephyr_file:`boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7_defconfig`` and
+:zephyr_file:`boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m4_defconfig`
+
+For more details please refer to `STM32H745-Disco UM`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- USART_3 TX/RX : PB10/PB11 (ST-Link Virtual Port Com)
+- USART_1 TX/RX : PB6/PB7 (Arduino Serial)
+- SPI_2 NSS/SCK/MISO/MOSI : PB4/PD3/PI2/PB15 (Arduino SPI)
+- I2C_4 SCL/SDA: PD12, PD13 (Arduino I2C)
+- USER_PB : PC13
+- LD1 : PI13
+- LD2 : PJ2
+- LD3 : PD3
+
+System Clock
+------------
+
+STM32H745I-DISCO System Clock could be driven by an internal or external
+oscillator, as well as the main PLL clock. By default, the System clock is
+driven by the PLL clock at 480MHz, driven by an 25MHz high-speed external clock.
+
+Serial Port
+-----------
+
+STM32H745I-DISCO board has 4 UARTs and 4 USARTs. The Zephyr console output is
+assigned to UART3. Default settings are 115200 8N1.
+
+Resources sharing
+-----------------
+
+The dual core nature of STM32H745 SoC requires sharing HW resources between the
+two cores. This is done in 3 ways:
+
+- **Compilation**: Clock configuration is only accessible to M7 core. M4 core only
+  has access to bus clock activation and deactivation.
+- **Static pre-compilation assignment**: Peripherals such as a UART are assigned in
+  devicetree before compilation. The user must ensure peripherals are not assigned
+  to both cores at the same time.
+- **Run time protection**: Interrupt-controller and GPIO configurations could be
+  accessed by both cores at run time. Accesses are protected by a hardware semaphore
+  to avoid potential concurrent access issues.
+
+Programming and Debugging
+*************************
+
+Applications for the ``stm32h745i_disco`` board should be built per core target,
+using either ``stm32h745i_disco/stm32h745xx/m7`` or ``stm32h745i_disco/stm32h745xx/m4`` as the target
+(see :ref:`build_an_application` and :ref:`application_run` for more details).
+
+.. note::
+
+   If using OpenOCD you will need a recent development version as the last
+   official release does not support H7 series and ST-LINK V3 yet.
+   Following links may be helpful: `OpenOCD installing Debug Version`_
+   and `OpenOCD installing with ST-LINK V3 support`_
```

Comment:
I don't think this is still applicable, need to be checked.

Reply:
The support for H7 was added in [0.11.0](https://github.com/openocd-org/openocd/blob/9ea7f3d647c8ecf6b0f1424002dfc3f4504a162c/NEWS-0.11.0#L63). Should this be removed?

Tested with OpenOCD 0.12.0 by `openocd -f boards/st/stm32h745i_disco/support/openocd.cfg `.

Reply:
> Should this be removed?

Yes

Reply:
Done